src/data/contactData.ts

```diff
@@ -0,0 +1,9 @@
+export interface Contacts {
+  firstname: string;
+  lastname: string;
```

Comment:
Still think it's better to just have a name field for the contact form -- we can avoid the gap issue and also not everyone has a last name.